src/facts/purple-coneflower.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Purple Coneflower
+image:
+  src: ../images/15264929397_9db643db03_k.jpg
```

Comment:
Please download the original image from Flickr instead of a resized one.

Reply:
Will do thanks Paul